modules/installation/pages/pubcloud-setup.adoc

```diff
@@ -2,17 +2,55 @@
 = Public Cloud Setup
 
 Public Cloud providers pre-install {productname}, so you do not need to perform any installation steps.
-However, {productname} Server needs to be registered with {scc} to receive updates before you can log in.
+However, you must have the correct {productname} and {sle} entitlements to receive updates before you can log in.
 
-For detailed instructions on registering {productname} to {scc}, see xref:installation:server-setup.adoc[].
+For most bring your own service (BYOS) images, these entitlements are provided by {suse}, and can be obtained using the {scc}.
+If you are using an on demand image, you need to do some extra preparation before you can register to {scc}.
 
-When you have registered, all {sle} modules will be activated.
-You will also need to activate the public cloud module.
+When you have registered, all {sle} modules are activated.
+You can then activate the public cloud module.
 
 
 
-.Procedure: Activating the Public Cloud Module
+.Procedure: Preparing On Demand Images
+. Log in the on demand instance.
+. At the command prompt, as root, remove the `cloud-regionsrv-client` package:
++
+----
+zypper rm cloud-regionsrv-client
+----
+. Remove additional packages specific to your cloud provider:
++
+* Amazon EC2:
++
+----
+zypper rm regionServiceClientConfigEC2 regionServiceCertsEC2
+----
++
+* Google Compute Engine:
++
+----
+zypper rm cloud-regionsrv-client-plugin-gce regionServiceClientConfigGCE regionServiceCertsGCE
+----
++
+* Microsoft Azure:
++
+----
+zypper rm regionServiceClientConfigAzure regionServiceCertsAzure
+----
+. Open the ``/etc/hosts`` file, and delete the entry that contains ``@susecloud.net``.
+. Remove the ``SUSEConnect`` configuration file:
++
+----
+rm /etc/SUSEConnect
+----
+
 
+For instructions on registering {productname} to {scc}, see xref:installation:server-setup.adoc[].
+
+
+
+.Procedure: Activating the Public Cloud Module
 . On the {productname} Server, open the {yast} management tool, and navigate to menu:Software[Software Repositories].
 . Click btn:[Add] and select [guimenu]``Extensions and Modules from Registration Server``.
 . In the [guimenu]``Available extensions`` field, select [guimenu]``Public Cloud Module``.
```

Comment:
I think this isn't a "field" but a "list".

Reply:
Well, it's a drop-down box, I guess, but doesn't "field" cover everything?